packages/components/src/placeholder/style.scss

```diff
@@ -74,6 +75,11 @@
 	margin-bottom: 1em;
 }
 
+.components-placeholder__error {
+	margin-top: 1em;
+	width: 100%;
```

Comment:
Required to render underneath the help text, considering the `display: flex` based wrapper.